source/languages/en/riak/dev/advanced/write-once.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Write Once Buckets
+project: riak
+version: 2.1.0+
+document: tutorial
+audience: intermediate
+keywords: [developers, buckets]
+---
+
+Riak 2.1.0 introduces the concept of "write once" buckets; buckets whose entries
+are intended to be written exactly once, and never updated or over-written.
+Buckets of this type have the special feature that they circumvent the normal
+"coordinated put" path, which would otherwise result in a read on the
+coordinating vnode before the write. Avoiding coordinated puts results in higher
+throughput and lower put latency, at the cost of different semantics in the
+degenerate case of sibling resolution.
+
+## Configuration
+
+We introduce the `write_once` bucket type property, which, when set to `true`,
+treats all key/value entries as semantically "write once"; once written, entries
+should not be modified or over-written by the user.
+
+The `write_once` property is a boolean property that is applied to a bucket type
+and may only be set at bucket creation time. Once a bucket type has been set
+with this property and activated, the `write_once` property may not be modified.
+
+The `write_once` property is incompatible with specification of [[Data
+Types|Using Data Types]] and [[Strong Consistency|Using Strong Consistency]].
+Any attempt to create a bucket type with the `write_once` property set to `true`
+and with data types or the strong consistency property set will fail.
+
+The `write_once` property may not be set on the default bucket type, and may not
+be set on individual buckets.
+
+The `lww` and `allow_mult` properties are ignored, if the `write_once` property
+is set to `true`.
+
+The following example shows how to configure a bucket type with the `write_once`
+property:
+
+```
+prompt$ riak-admin bucket-type create my-bucket-type '{"props": {"write_once": true}}'
+my-bucket-type created
+
+prompt$ riak-admin bucket-type activate my-bucket-type
+my-bucket-type has been activated
+
+prompt$ riak-admin bucket-type status my-bucket-type
+my-bucket-type is active
+...
+write_once: true
+...
+```
+
+## Quorum
+
+The write once path supports the `W`, `PW`, and `DW` configuraton values.
```

Comment:
Changed here to 

```
The write path used by Write Once Buckets supports the `W`, `PW`, and `DW`
configuration values. However, if `DW` is specified, then the value of `W` is
taken to be the maximum of the `W` and `DW` values. For example, for `n_val 3`,
if `DW` is set to `all`, then `W` will be `3`.

This write additionally supports the `sloppy_quroum` property. If set to
`false`, only primary nodes will be selected for calculation of write quorum
nodes.
```

I'll do my best to remove instances of `write once` and `"write once"`, so that we're left with the Strong Case concept of `Write Once Buckets`, and the actual `"\`write_once`\" configuration option.